includes/fraud-prevention/class-fraud-risk-tools.php

```diff
@@ -202,7 +202,7 @@ public static function get_high_protection_settings() {
 				Check::check(
 					'order_total',
 					Check::OPERATOR_GT,
-					1000 * 100
+					implode( '|', [ 1000 * 100, 'usd' ] )
```

Comment:
Same as the comment I wrote for the standard protection defaults. (Probably that comment should be here and this on the other one 😁 )

Reply:
Thanks for your comment. It was addressed on [2deffc6](https://github.com/Automattic/woocommerce-payments/pull/5926/commits/2deffc6786e11e09449955197b698a19e4c4c6d7).